patricktunez123/Andela_Teamwork
# #168895744 API endpoints code refactoring

#### What does this PR do?
Have all same data for testing in mockData JavaScript file to keep test DRY.
Have try and catch logic 
Add ES6 style
#### Description of Task to be completed?
Have the whole app in ES6
Have try and catch logic
Have DRY tests
#### How should this be manually tested?
After cloning the project, cd into it and run npm install, npm run start and then test every endpoint in postman, every end point should  be  working
#### What are the relevant pivotal tracker stories?
#168895744
